esi/lib/nodes.py

```diff
@@ -61,7 +103,7 @@ def network_list(connection, filter_node=None, filter_network=None):
         'network_info': [
             {
                 'baremetal_port': openstack.baremetal.v1.port.Port,
-                'network_port': [openstack.network.v2.port.Port] or [],
+                'network_ports': [openstack.network.v2.port.Port] or [],
```

Comment:
Does this effect python-esiclient?

Reply:
This does not affect python-esiclient, just a documentation error